README.md

```diff
@@ -104,3 +104,18 @@ adding the path to the theme's logo.
 Add the following to the root URLConf:
 
 `handler500 = 'dc_theme.urls.dc_server_error'`
+
+## HTML testing page
+
+To test how HTML elemtnts are likely to render in your project, this theme provides a tester URL. Enable it by adding the following to your root url config:
+
+```
+urlpatterns += [
+    url(r'^dc_base_theme', include('dc_theme.urls')),
+]```
```

Comment:
The end of this fenced code block needs to be on the next line down, otherwise it renders weird:

![screenshot at 2018-03-28 13 41 11](https://user-images.githubusercontent.com/6025893/38029433-c1220f54-328d-11e8-9c6a-3f44a6bbe6d6.png)

Reply:
That'll teach me for using the GutHub interface for PRs ,but not using the preview tab!